lib/onebox/helpers.rb

```diff
@@ -169,15 +148,9 @@ def self.blank?(value)
     end
 
     def self.truncate(string, length = 50)
-      string = sanitize(string)
       string.size > length ? string[0...(string.rindex(" ", length) || length)] + "..." : string
     end
 
-    def self.title_attr(meta)
```

Comment:
Please make sure that `Helpers.title_attr` isn't used anymore. Maybe replace it with `og.title_attr` wherever it's still used?